go/types/rolling_value_hasher.go

```diff
@@ -35,8 +35,8 @@ func chunkingConfig() (pattern, window uint32) {
 func smallTestChunks() {
 	chunkConfigMu.Lock()
 	defer chunkConfigMu.Unlock()
-	chunkPattern = uint32(1<<7 - 1) // Avg Chunk Size of 128 bytes
-	chunkWindow = uint32(32)
+	chunkPattern = uint32(1<<8 - 1) // Avg Chunk Size of 256 bytes
```

Comment:
i had to increase these to avoid the infinite chunking bug when in "smallTestChunks" mode.

one side effect of this is that we are once again chunking over type bytes in refs.